src/gc/impl/conservative/gc.d

```diff
@@ -2596,12 +2670,65 @@ struct Gcx
             prepTime += (stop - start);
             start = stop;
 
-            version (COLLECT_PARALLEL)
-                bool doParallel = config.parallel > 0;
-            else
-                enum doParallel = false;
-
-            if (doParallel)
+            // Forking is enabled, so we fork() and start a new concurrent mark phase
+            // in the child. If the collection should not block, the parent process
+            // tells the caller no memory could be recycled immediately (if this collection
+            // was triggered by an allocation, the caller should allocate more memory
+            // to fulfill the request).
+            // If the collection should block, the parent will wait for the mark phase
+            // to finish before returning control to the mutator,
+            // but other threads are restarted and may run in parallel with the mark phase
+            // (unless they allocate or use the GC themselves, in which case
+            // the global GC lock will stop them).
+            if (doFork)
```

Comment:
This section, too.

Reply:
Done, see commit 
https://github.com/dlang/druntime/pull/2604/commits/d3fe6ad574d09c8facc70172f9162b1d3c256fc6